### DISCUSSION TITLE
Display operating costs in buy menu of trains and vehicles

### DISCUSSION THREAD

**Discussion by zwonk32** [voxeltycoon/issues#882] [General]
Display operating costs in buy menu of trains and vehicles

**Comment by DanielBarreleiro** [comment]
@zwonk32 You mean like this, that already exists?
![image](https://user-images.githubusercontent.com/47695358/128683543-a6a671e4-4a2a-4d23-93be-3ce1f5fe6c51.png)
![image](https://user-images.githubusercontent.com/47695358/128683597-2fb21890-165b-473f-8c5e-405868d45370.png)

**Comment by zwonk32** [comment]
No... I mean in the Windows where i chose the vehicles and can compare them

⁣BlueMail for Android herunterladen ​

Am 9. Aug. 2021, 11:10, um 11:10, Daniel Barreleiro ***@***.***> schrieb:
***@***.*** You mean like this, that already exists?
